src/neo/Cryptography/MPT/MPTTrie.Get.cs

```diff
@@ -32,8 +33,8 @@ private bool TryGet(ref MPTNode node, ReadOnlySpan<byte> path, out ReadOnlySpan<
                 case HashNode hashNode:
                     {
                         if (hashNode.IsEmpty) break;
-                        var newNode = Resolve(hashNode);
-                        if (newNode is null) break;
+                        var newNode = Resolve(hashNode.Hash);
+                        if (newNode is null) throw new KeyNotFoundException("Internal error, can't resolve hash when mpt get");
```

Comment:
```suggestion
                        if (newNode is null) return false;
```
Change to this?

Reply:
Done. We should throw exception. It's a structure error. I change the exception to `InvalidOperationException`.